assets/src/domain/eventEditor/ui/datetimes/datesList/index.tsx

```diff
@@ -5,9 +5,9 @@ import AddNewDateButton from './AddNewDateButton';
 import List from './List';
 import useDatetimes from '../../../services/apollo/queries/datetimes/useDatetimes';
 import { useStatus, TypeName } from '../../../../../application/services/apollo/status';
-import EmptyState from '../../../../../application/ui/display/EmptyState';
-import ErrorIndicator from '../../../../../application/ui/display/ErrorIndicator';
-import LoadingIndicator from '../../../../../application/ui/display/LoadingIndicator';
+import EmptyState from '@appDisplay/EmptyState';
+import ErrorIndicator from '@appDisplay/ErrorIndicator';
+import LoadingIndicator from '@appDisplay/LoadingIndicator';
```

Comment:
These could be exported/imported via an index file

Reply:
Yup, good point: https://github.com/eventespresso/event-espresso-core/pull/2366/commits/bf55ed1b86eb54dedce0ed1d189d9c15ab7545f2

Reply:
the only thing I don't like is the `index` from `@appDisplay/index`, but not a biggie